chore: deprecate handwritten code

Neenu1995 opened the pull request.

Neenu1995 committed:
chore: deprecate handwritten code

gcf-owl-bot[bot] committed:
Merge branch 'branch-deprecation' of https://github.com/googleapis/java-resourcemanager into branch-deprecation

renovate-bot committed:
chore(deps): update dependency com.google.cloud:google-cloud-resourcemanager to v0.119.1-alpha (#438)

yoshi-automation committed:
chore: regenerate README (#442)

This PR was generated using Autosynth. :rainbow:


<details><summary>Log from Synthtool</summary>

```
2021-04-14 01:02:59,862 synthtool [DEBUG] > Executing /root/.cache/synthtool/java-resourcemanager/.github/readme/synth.py.
On branch autosynth-readme
nothing to commit, working tree clean
2021-04-14 01:03:01,096 synthtool [DEBUG] > Wrote metadata to .github/readme/synth.metadata/synth.metadata.

```
</details>

Full log will be available here:
https://source.cloud.google.com/results/invocations/736fffb9-53c4-4214-9648-b19bfc5fa6a2/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

chingor13 commented on google-cloud-resourcemanager/src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java:
```diff
@@ -27,7 +27,9 @@
  *
  * @see <a
  *     href="https://cloud.google.com/resource-manager/reference/rest/v1/ListAvailableOrgPolicyConstraintsResponse#Constraint">Constraint</a>
+ * @deprecated v3 GAPIC client of ResourceManager is now available
```
For the best developer experience, let's try and find the associated model/rpc in the v3 client

chingor13 reviewed (changes requested):
If possible, we want each `@deprecated` javadoc entry to point to the replacement class/method.

Also, we want this to appear in the release notes so it should not be a `chore` - perhaps a `feat`?

gcf-owl-bot[bot] committed:
build: update .OwlBot.lock with new version of post-processor (#443)

This PR updates the docker container used for OwlBot. This container performs post-processing tasks when pull-requests are opened on your repository, such as:

* copying generated files into place.
* generating common files from templates.

Version sha256:b3c6a25dc0b7474ef211b01aac777f24efa3cb97ef04d22f886b2b78f55744d8 was published at 2021-04-14T16:24:29.580Z.

renovate-bot committed:
build(deps): update dependency com.google.cloud:google-cloud-shared-config to v0.11.1 (#446)

[![WhiteSource Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Change | Age | Adoption | Passing | Confidence |
|---|---|---|---|---|---|
| [com.google.cloud:google-cloud-shared-config](https://togithub.com/googleapis/java-shared-config) | `0.11.0` -> `0.11.1` | [![age](https://badges.renovateapi.com/packages/maven/com.google.cloud:google-cloud-shared-config/0.11.1/age-slim)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://badges.renovateapi.com/packages/maven/com.google.cloud:google-cloud-shared-config/0.11.1/adoption-slim)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://badges.renovateapi.com/packages/maven/com.google.cloud:google-cloud-shared-config/0.11.1/compatibility-slim/0.11.0)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://badges.renovateapi.com/packages/maven/com.google.cloud:google-cloud-shared-config/0.11.1/confidence-slim/0.11.0)](https://docs.renovatebot.com/merge-confidence/) |

---

### Release Notes

<details>
<summary>googleapis/java-shared-config</summary>

### [`v0.11.1`](https://togithub.com/googleapis/java-shared-config/blob/master/CHANGELOG.md#&#8203;0111-httpswwwgithubcomgoogleapisjava-shared-configcomparev0110v0111-2021-04-14)

[Compare Source](https://togithub.com/googleapis/java-shared-config/compare/v0.11.0...v0.11.1)

</details>

---

### Configuration

:date: **Schedule**: At any time (no schedule defined).

:vertical_traffic_light: **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

:recycle: **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

:no_bell: **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box.

---

This PR has been generated by [WhiteSource Renovate](https://renovate.whitesourcesoftware.com). View repository job log [here](https://app.renovatebot.com/dashboard#github/googleapis/java-resourcemanager).

renovate-bot committed:
chore(deps): update dependency com.google.cloud:libraries-bom to v20.1.0 (#450)

[![WhiteSource Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

This PR contains the following updates:

| Package | Change | Age | Adoption | Passing | Confidence |
|---|---|---|---|---|---|
| [com.google.cloud:libraries-bom](https://togithub.com/GoogleCloudPlatform/cloud-opensource-java) | `20.0.0` -> `20.1.0` | [![age](https://badges.renovateapi.com/packages/maven/com.google.cloud:libraries-bom/20.1.0/age-slim)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://badges.renovateapi.com/packages/maven/com.google.cloud:libraries-bom/20.1.0/adoption-slim)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://badges.renovateapi.com/packages/maven/com.google.cloud:libraries-bom/20.1.0/compatibility-slim/20.0.0)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://badges.renovateapi.com/packages/maven/com.google.cloud:libraries-bom/20.1.0/confidence-slim/20.0.0)](https://docs.renovatebot.com/merge-confidence/) |

---

### Configuration

:date: **Schedule**: At any time (no schedule defined).

:vertical_traffic_light: **Automerge**: Disabled by config. Please merge this manually once you are satisfied.

:recycle: **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

:no_bell: **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] <!-- rebase-check -->If you want to rebase/retry this PR, check this box.

---

This PR has been generated by [WhiteSource Renovate](https://renovate.whitesourcesoftware.com). View repository job log [here](https://app.renovatebot.com/dashboard#github/googleapis/java-resourcemanager).

yoshi-automation committed:
chore: regenerate README (#453)

This PR was generated using Autosynth. :rainbow:


<details><summary>Log from Synthtool</summary>

```
2021-04-19 14:46:36,664 synthtool [DEBUG] > Executing /root/.cache/synthtool/java-resourcemanager/.github/readme/synth.py.
On branch autosynth-readme
nothing to commit, working tree clean
2021-04-19 14:46:37,958 synthtool [DEBUG] > Wrote metadata to .github/readme/synth.metadata/synth.metadata.

```
</details>

Full log will be available here:
https://source.cloud.google.com/results/invocations/9a701fb1-d89d-4626-be50-3c54e70783d5/targets

- [ ] To automatically regenerate this PR, check this box. (May take up to 24 hours.)

Neenu1995 committed:
chore: redirect deprecated methods to alternatives in javadoc

gcf-owl-bot[bot] committed:
Merge branch 'branch-deprecation' of https://github.com/googleapis/java-resourcemanager into branch-deprecation

gcf-owl-bot[bot] committed:
🦉 Updates from OwlBot

gcf-owl-bot[bot] committed:
🦉 Updates from OwlBot
